src/com/xilinx/rapidwright/design/xdc/ConstraintTools.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2025, Advanced Micro Devices, Inc.
+ * All rights reserved.
+ *
+ * Author: Coherent Ho, Synopsys, Inc.
+ *
+ * This file is part of RapidWright.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.xilinx.rapidwright.design.xdc;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.xilinx.rapidwright.design.Design;
+import com.xilinx.rapidwright.design.blocks.PBlock;
+import com.xilinx.rapidwright.design.ConstraintGroup;
+
+/**
+ * A collection of methods to access design constraints.
+ *
+ * Created on: Oct 31, 2025
+ */
+public class ConstraintTools {
+    public Map<String, PBlock> getPBlockFromXDCConstraints(Design d){
```

Comment:
```suggestion
    public static Map<String, PBlock> getPBlockFromXDCConstraints(Design d) {
```
We want to make this method static so that users don't have to create an instance of `ConstraintTools`, when the method is static, the method can be called directly `ConstraintTools.getPBlockFromXDCConstraints()`.